code/datums/storage/subtypes/internal.dm

```diff
@@ -81,20 +81,14 @@
 /datum/storage/internal/webbing/New(atom/parent)
 	. = ..()
 	set_holdable(
-		can_hold_list = list(
-			/obj/item/stack/razorwire,
-			/obj/item/stack/sheet,
-			/obj/item/stack/sandbags,
-			/obj/item/stack/snow,
-			/obj/item/cell/lasgun/plasma,
-		),
```

Comment:
This is actually just a bug that needs fixing. Big webbing had those things in the CANT store list... xander presumably just typo'd it and make it CAN store by accident.